Sample Solutions/Data Lake/MGDC/src/JsonToCDM_CSV_SynapseNotebook.ipynb

```diff
@@ -0,0 +1,404 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "source": [
+        "# # #Initializing for test purposes\r\n",
+        "\r\n",
+        "# Define the variables used for creating connection strings\r\n",
+        "# initialize the blank variables with correct values\r\n",
+        "# Application (Client) ID\r\n",
+        "# AppId = \"\"\r\n",
+        "# # Application (Client) Secret Key\r\n",
+        "# AppKey = \"\"\r\n",
+        "# # Directory (Tenant) ID\r\n",
+        "# TenantId = \"\"\r\n",
+        "# StorageAccountName = \"\"\r\n",
+        "# PipelineId = \"\"\r\n",
+        "\r\n",
+        "AppId = \"e5217431-1131-41ef-8946-488cd0fae886\"\r\n",
+        "# Application (Client) Secret Key\r\n",
+        "AppKey = \".Bj7Q~pSloqDMckFBxm1Cfgdaz9KbE0wVd6o0\"\r\n",
+        "# Directory (Tenant) ID\r\n",
+        "TenantId = \"bd382938-3ba0-469f-8b45-fa06740d181f\"\r\n",
+        "StorageAccountName = \"mgdcvivadatalake\"\r\n",
+        "PipelineId = \"93d30026-56bb-4401-bf83-76d91521f404\"\r\n",
+        "\r\n",
+        "UserDatasetFolder = \"user\"\r\n",
+        "ManagerDatasetFolder = \"manager\"\r\n",
+        "MessageDatasetFolder = \"message\"\r\n",
+        "EventDatasetFolder = \"event\"\r\n",
+        "TeamsDatasetFolder = \"teams\"\r\n",
+        "\r\n",
+        "OfficeDataFileSystem = \"json\"\r\n",
+        "CdmDataFileSystem = \"cdm\"\r\n",
+        "CdmModelName = \"O365-data\"\r\n",
+        "CsvDataFileSystem = \"csv\""
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "tags": [
+          "parameters"
+        ]
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import sys\r\n",
+        "import json\r\n",
+        "\r\n",
+        "from pyspark import SparkContext, SparkConf\r\n",
+        "from pyspark.sql import SQLContext\r\n",
+        "from pyspark.sql.functions import *\r\n",
+        "from pyspark.sql.types import StringType\r\n",
+        "\r\n",
+        "# constants\r\n",
+        "inputFilePath = \"abfss://{}@{}.dfs.core.windows.net/{}/{}/????????\\-*\"\r\n",
+        "storageAccount = \"{}.dfs.core.windows.net\"\r\n",
+        "outputFilePath = \"https://{}.dfs.core.windows.net/{}/{}\"\r\n",
+        "csvOutputPath = \"abfss://{}@{}.dfs.core.windows.net/{}/\"\r\n",
+        "manifestPath = \"{}/{}/default.manifest.cdm.json\"\r\n",
```

Comment:
will this be documented around or defined in another area?

Reply:
These will be formatted/completed by the pipeline parameter values. But will add some comment on the file